corehq/apps/translations/app_translations/download.py

```diff
@@ -23,33 +23,41 @@
 
 def get_bulk_app_single_sheet_by_name(app, lang):
     rows = []
-    for module_index, module in enumerate(app.modules):
+    for module in app.modules:
         prefix = [get_module_sheet_name(module)]
 
         # Name / menu media row
         # Leave blanks for case_property, list_or_detail, and label
-        rows.append(prefix + ['', '', ''] + get_menu_row([module.name.get(lang)],
-                                                         [module.icon_by_language(lang)],
-                                                         [module.audio_by_language(lang)]))
+        rows.append(
+            prefix + ['', '', ''] +
+            get_menu_row([module.name.get(lang)],
+                         [module.icon_by_language(lang)],
+                         [module.audio_by_language(lang)]) +
+            [module.unique_id]
+        )
 
         # Detail case properties, etc.
         for row in get_module_rows([lang], module):
             # Insert blank for label
             rows.append(prefix + list(row[:2]) + [''] + list(row[2:]))
 
-        for form_index, form in enumerate(module.forms):
+        for form in module.forms:
             # Leave blanks for case_property and list_or_detail
             prefix = [get_form_sheet_name(form), '', '']
 
             # Name / menu media row
             # Leave another blank for label
-            rows.append(prefix + [''] + get_menu_row([form.name.get(lang)],
-                                                     [form.icon_by_language(lang)],
-                                                     [form.audio_by_language(lang)]))
+            rows.append(
+                prefix + [''] +
+                get_menu_row([form.name.get(lang)],
+                             [form.icon_by_language(lang)],
+                             [form.audio_by_language(lang)]) +
+                [form.unique_id]
+            )
 
             # Questions
             for row in get_form_question_rows([lang], form):
-                rows.append(prefix + row)
+                rows.append(prefix + row + [''])
```

Comment:
This adds an empty string for unique id for form question rows, but the similar rows for modules (the loop over the `get_module_rows` up on new lines 40-42) don't get an empty string. Should they?

Reply:
It's a little odd. It looks like form rows have the same number of columns as the headers, but module rows don't. Not sure why. I'll make them all the same.

Reply:
Turns out module case list & case detail rows just didn't bother adding empty strings for media. I've made all the single-sheet rows the same length fwiw.